contribute/style-guides/redux.md

```diff
@@ -117,17 +117,48 @@ Type '{ nonExistingProperty: string; data: string[]; }' is not assignable to typ
   Object literal may only specify known properties, and 'nonExistingProperty' does not exist in type 'ExampleReducerState'. [2322]
 ```
 
-## New test functionality
+## Test functionality
 
 ### reducerTester
 
 Fluent API that simplifies the testing of reducers
 
-#### Example
+#### Simple Example
```

Comment:
```suggestion
#### Usage
```

Reply:
I like this better, thanks

Reply:
I would go with `Usage` everywhere when `[Simple|Complex] example` is used